index.html

```diff
@@ -115,24 +150,128 @@ <h3>About us</h3>
 
         </div>
     </div>
+<!--discount-->
     <div class="container">
-        <div class="row ">
+        <div class="row discount">
+            <div class=" col-12 col-6 about-img mob-img">
+                <img src="/img/tree.jpeg" alt="">
+            </div> 
             <div class="col-12 col-6 about">
                 <h1>BUY ONLINE <br> 
                     NOW & GET<br>
                      10% OFF!</h1>
             </div>
+        </div>
+    </div>
+<!--form-->
+    <div class="container section-form">
+        <h2>Contact Us</h2>
+        <div class="row ">
+           
+            <form>
+                <div class="row">
+                    <div class="col-12 xl-col-6 name-form">
+                        <label for="fname">First name*</label><br>
+                        <input type="text" id="fname" name="fname" value="Your name">
+                    </div>
+                    <div class="col-12 xl-col-6 email-form">
+                        <label for="lname">Your email*</label><br>
+                        <input type="email" id="email" name="email" value="Your email">
+                    </div>
+                 </div>
+                <div class="row">
+                <div class="col-12 xl-col-10 email-form">
+                    <label for="lname">Your phone*</label><br>
+                    <input type="tel" id="phone" name="phone" value="Your phone">
+                </div>
+                <div class="col-12 xl-col-10 email-form">
+                    <label for="lname">Enter your message here*</label><br>
+                    <textarea class="article-input" id="article-input" type="text" rows="9" >Message</textarea> 
+                    <input class="sub-botton" type="submit" value="Send">
+                </div>
+            </div>
+                
+              </form>
+        </div>
+
+    </div>
+
+<!--Footer-->
+    <div class="container section-footer">
+        <div class="row">
+
+            <div class=" box col-12 xl-col-3 ">
+                <div >
+                    <h3>Our Store</h3>
```

Comment:
You reference h3 but don't include it in the css? if it is used often could be worthwhile to add your own CSS for it